enterprise/management/commands/update_role_assignments_with_customers.py

```diff
@@ -141,7 +141,7 @@ def _handle(self, batch_size, role_name, enterprise_customer_uuid=None, dry_run=
             customers_by_user_id = self._get_customers_by_user_id(ecu_ids_batch)
 
             assignments_by_user_id_role = self._get_role_assignments_by_user_id_and_role(
-                customers_by_user_id.keys(),
+                customers_by_user_id.keys(),  # pylint: disable=dict-keys-not-iterating
```

Comment:
I think one of these "`.keys()` as a method parameter" had a sub method that still was not iterating through the keys, but was making a set from them, so it seemed fine to leave be.